Added note about using python in code nodes (#1856)
n8n-io/n8n-docs: docs/courses/level-one/chapter-5/chapter-5.5.md

<figure><img src="/_images/courses/level-one/chapter-five/l1-c5-5-5-code-node.png" alt="Code node" style="width:100%"><figcaption align = "center"><i>Code node</i></figcaption></figure>

/// note | Using Python in code nodes

> The Code node also allows you to use Python. Learn more about how to this, see the [Code node](/integrations/builtin/core-nodes/n8n-nodes-base.code/){:target="_blank" .external} documentation.

Reviewer: ```suggestion
You can also use Python in the Code node. To learn more about this, refer to the [Code](/code/code-node/) documentation.
```
